plugins/CMakeLists.txt

```diff
@@ -18,3 +18,4 @@ INCLUDE_DIRECTORIES(
 FOREACH(PLUGIN ${PLUGIN_LIST})
 	ADD_SUBDIRECTORY(${PLUGIN})
 ENDFOREACH()
+ADD_SUBDIRECTORY("SlicerT")
```

Comment:
It should be added to plugin list after the Sid emulation synth here: https://github.com/LMMS/lmms/blob/9b1933e544697bfa67d337ed12e0501d66a66ec0/cmake/modules/PluginList.cmake#L61

Reply:
done